codeship-steps.yml

```diff
@@ -1,6 +1,6 @@
 - name: "Test"
   service: test
-  command: composer test
+  command: test_reporter
```

Comment:
Does this always run composer test? Can we run custom commands through it too?
I've found it helpful for local testing to be able to run just a specific test so if we can preserve that it'd be really helpful